cockroachcloud/cloud-sso.md

```diff
@@ -73,7 +73,9 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
 
 ### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
 
-Enterprise Authentication supports only a flow initiated by {{ site.data.products.db }}. Initiation from the IdP is not supported.
+The primary flow is to initiate {{ site.data.products.db }} SSO through the cloud console UI.
+
+Initiation from the IdP is also possible, but not supported through the cloud console. Contact your account team for more information.
```

Comment:
I wonder if it makes sense to rephrase this as "If you need to initiate SSO integration from the IdP, contact your account team for assistance."